compiler/qsc_parse/src/lex/cooked.rs

```diff
@@ -139,7 +148,7 @@ pub(crate) enum TokenKind {
     /// `;`
     Semi,
     /// A string literal.
-    String(StringToken),
+    StringToken(StringToken),
```

Comment:
Why is this renamed?

Reply:
Agreed. I'd like to keep any refactorings like this to an isolated PR.

Reply:
Though I don't like * imports unless there is a real prelude being imported.

Reply:
I've reversed the rename.